src/search/FindBar.js

```diff
@@ -444,6 +444,7 @@ define(function (require, exports, module) {
         this.searchField = new QuickSearchField(searchFieldInput, {
             verticalAdjust: searchFieldInput.offset().top > 0 ? 0 : this._modalBar.getRoot().outerHeight(),
             maxResults: 20,
+            shouldHighLightFirstIndex: false,
```

Comment:
Oh sorry for the vague comment, I meant to explicitly add it as `shouldHighLightFirstIndex: true` in the Quick Open so it's defined in both.